src/test/java/dev/kodice/hexcolorgenerator/models/dto/RandomColorsResponseTest.java

```diff
@@ -0,0 +1,30 @@
+package dev.kodice.hexcolorgenerator.models.dto;
+
+import static com.google.code.beanmatchers.BeanMatchers.hasValidBeanConstructor;
+import static com.google.code.beanmatchers.BeanMatchers.hasValidBeanToString;
+import static com.google.code.beanmatchers.BeanMatchers.hasValidGettersAndSetters;
+import static com.google.code.beanmatchers.BeanMatchers.registerValueGenerator;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.time.LocalDateTime;
+
+import org.junit.jupiter.api.Test;
+
+class RandomColorsResponseTest {
+
+	@Test
+	public void testRandomColorsResponse() {
+		registerValueGenerator(() -> {
+			double m = (Math.random() * (12)) + 1;
```

Comment:
Qué te parece cambiar la variable `m` por `month`
```suggestion
			double month = (Math.random() * (12)) + 1;
```